src/metabase/db/spec.clj

```diff
@@ -26,7 +27,8 @@
     :subprotocol                   "postgresql"
     :subname                       (make-subname host (or port 5432) db)
     ;; I think this is done to prevent conflicts with redshift driver registering itself to handle postgres://
-    :OpenSourceSubProtocolOverride true}
```

Comment:
https://jdbc.postgresql.org/documentation/94/connect.html see docs for `ApplicationName`

Reply:
I wonder if it would be possible/useful to get the DW DB ID into here (or maybe just use `app` if it's the app DB)?  That would help narrow down between different expected pools for different DBs.

Reply:
Adding this is a little trickier than I thought because the code pathway isn't ` Database`-ID-aware -- it's basically connection properties -> JDBC spec (it's used for testing whether we can connect to the DB before we even save a ` Database` for example) -- so adding this info would require a little bit of refactoring. I don't really want to hold up the 39 release to do that, so let's do that as a separate 39.1 or 40 follow-on PR

Reply:
Opened #15679 for that follow-up.